bootstrap.json

```diff
@@ -5,12 +5,13 @@
             "github.com/fzipp/gocyclo",
             "github.com/gordonklaus/ineffassign",
             "github.com/client9/misspell/cmd/misspell",
-            "github.com/golang/dep/cmd/dep",
             "golang.org/x/mobile/cmd/gomobile",
             "golang.org/x/tools/go/ast/astutil",
             "golang.org/x/tools/go/internal",
             "golang.org/x/tools/go/gcexportdata",
             "github.com/golangci/golangci-lint",
+            "github.com/frapposelli/wwhrd",
+            "github.com/clamoriniere/modvendor",
```

Comment:
curious, why not upstream `github.com/goware/modvendor`?

Reply:
They have introduced an incompatibility in the current "master" branch with go1.13 and when we overwrite the "source" and the "version" for a dependency.
https://github.com/goware/modvendor/commit/69c7309340cf691a297ac139b2c2682e45da1e66
Since now I know the problem I will move back "github.com/goware/modvendor" using the commit before this change

Reply:
done